pywren/serialize/cloudpickle/cloudpickle.py

```diff
@@ -12,7 +12,7 @@
 <https://web.archive.org/web/20140626004012/http://www.picloud.com/>`_.
 
```

Comment:
I think we will sync their more recent changes; is there a way to tell pylint to ignore certain files? In the interest of getting this out there I think we should merge this PR asap.